### PR TITLE
cleanup + tests

### PR DESCRIPTION
**Pull request by pfitzseb**
I _think_ we don't need those imports, but not sure -- can you have a look @MikeInnes?

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,7 +1,4 @@
 julia 0.5-
 Lazy
-LNR
-JuliaParser
```
Comment (MikeInnes) [inline REQUIRE:4]: I guess these are vestiges from when we did scoping here, so it should be safe to remove them if they're not obviously used.

File: src/module.jl [hunk 2]
```diff
@@ -2,8 +2,6 @@
 # Some file utils
 # –––––––––––––––
 
-VERSION > v"0.4-" && (beginswith = Base.startswith)
```
Comment (MikeInnes) [inline src/module.jl:5]: Just as a cleanup, there are a few places where `Base.startswith` can be unqualified

**Review by MikeInnes — CHANGES_REQUESTED** [review]
Looks good, would just like to see the startswith change. This new review interface is quite nice isn't it?

**Comment by pfitzseb** [comment]
Yeah, the review interface seems pretty cool... gotta have to try it sometime :D

**Comment by MikeInnes** [comment]
Oh, can you run a julia-client test with this change as well?

**Review by MikeInnes — APPROVED** [review]

**Comment by pfitzseb** [comment]
I can, but they're broken on Windows right now ;) Will have a look at fixing them though...

**Comment by pfitzseb** [comment]
julia-client specs pass, I'll merge this.

**Pull request merged**